app/tiebout.hoon

```diff
@@ -0,0 +1,253 @@
+::
+::::  /app/tiebout/hoon
+::
+/?  309
+/-  hall
+/+  tiebout
+=,  tiebout
```

Comment:
I learned recently that you can do `/+  *tiebout` to not put a face on it, so you don't need to `=,` afterwards.

Reply:
cool!